libflux/core/src/semantic/convert.rs

```diff
@@ -242,42 +242,311 @@ fn convert_monotype(
+                                },
+                                tail: MonoType::Record(Box::new(types::Record::Extension {
+                                head: types::Property {
+                                    k: "<".to_string(),
+                                    v: MonoType::Var(new_tvar),
+                                },
+                                tail: MonoType::Record(Box::new(types::Record::Extension {
+                                head: types::Property { 
+                                    k: ">=".to_string(),
+                                    v: MonoType::Var(new_tvar),
+                                },
+                                tail: MonoType::Record(Box::new(types::Record::Extension {
+                                head: types::Property {
+                                    k: "<=".to_string(),
+                                    v: MonoType::Var(new_tvar),
+                                },
+                                tail: rec
+                            }))
+                            }))
+                            }))
+                            }))
+                        },
+                        "Equatable" => {
+                            rec = MonoType::Record(Box::new(types::Record::Extension {
+                                head: types::Property {
+                                    k: "==".to_string(),
+                                    v: MonoType::Var(new_tvar),
+                                },
+                                tail: MonoType::Record(Box::new(types::Record::Extension {
+                                head: types::Property {
+                                    k: "!=".to_string(),
+                                    v: MonoType::Var(new_tvar),
+                                },
+                                tail: rec
+                            }))
+                            }))
+                        },
+                        "Nullable" => {
+                            rec = MonoType::Record(Box::new(types::Record::Extension {
+                                head: types::Property {
+                                    k: "==".to_string(),
+                                    v: MonoType::Var(new_tvar),
+                                },
+                                tail: MonoType::Record(Box::new(types::Record::Extension {
+                                head: types::Property {
+                                    k: "!=".to_string(),
+                                    v: MonoType::Var(new_tvar),
+                                },
+                                tail: rec
+                            }))
+                            }))
+                        },
+                        "Negatable" => {
+                            rec = MonoType::Record(Box::new(types::Record::Extension {
+                                head: types::Property {
+                                    k: "~".to_string(),
```

Comment:
How about we change this to `@Neg` instead?

Reply:
Yup, completely agree.